e2e/e2etests/test_stress_sui_deposit.go

```diff
@@ -0,0 +1,43 @@
+package e2etests
+
+import (
+	"cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+)
+
+// TestStressSuiDeposit tests the stressing deposit of SUI
+func TestStressSuiDeposit(r *runner.E2ERunner, args []string) {
+	require.Len(r, args, 2)
+
+	amount := utils.ParseBigInt(r, args[0])
+	numDeposits := utils.ParseInt(r, args[1])
+
+	r.Logger.Print("starting stress test of %d SUI deposits", numDeposits)
+
+	// create a wait group to wait for all the deposits to complete
+	var eg errgroup.Group
+
+	// send the SUI deposit transactions
+	for i := range numDeposits {
+		resp := r.SuiDepositSUI(r.EVMAddress(), math.NewUintFromBigInt(amount))
```

Comment:
I know that this was fixed, but let's be verbose to indicate that we reuse `i` across goroutines.

https://go.dev/blog/loopvar-preview
 
```suggestion
    i := i
		resp := r.SuiDepositSUI(r.EVMAddress(), math.NewUintFromBigInt(amount))
```

Reply:
https://github.com/zeta-chain/node/pull/3893/commits/441eb12f1047202f18b645566c83c08ad8709b34